src/openrct2/rct12/RCT12.h

```diff
@@ -589,9 +589,9 @@ assert_struct_size(RCT12EntranceElement, 8);
 struct RCT12BannerElement : RCT12TileElementBase
 {
 private:
-    uint8_t index;    // 4
-    uint8_t position; // 5
-    uint8_t flags;    // 6
+    uint8_t index;        // 4
+    uint8_t position;     // 5
+    uint8_t banner_flags; // 6
```

Comment:
You will need to check every instance of RCT12BannerElement to make sure if it was meant to use banner_flags or flags.

Reply:
Luckily that member is private, so the changes were easily done.

Reply:
I question if these should even be called flags maybe should call it allowed edges. Give me a little while to check things

Reply:
after a review of the codebase I dont think these should be called flags. They are `AllowedEdges` can you please rename them. Can you also rename the `BannerElement` version of them as well `TileElement.h` will need modifcations in `Banner.cpp` as well. I realise that `AllowedEdges` wont match the code style surrounding this but that is how it should be styled as per our coding style document

Reply:
That totally makes sense.